src/stories/Jobs/CanCloseJob/endpoint.spec.js

```diff
@@ -0,0 +1,43 @@
+const contextClassRef = requireUtil("contextHelper");
+const randomUser = requireUtil("randomUser");
+const knex = requireKnex();
+const httpServer = requireHttpServer();
+const JobsRepo = requireRepo("jobs");
+
+describe("Test API CanCloseJob", () => {
+  beforeAll(async () => {
+    contextClassRef.user = randomUser();
```

Comment:
can we also truncate the tables so that the test DB doesn't have any data?

Reply:
ok